src/data/mockDecks.ts

```diff
@@ -0,0 +1,46 @@
+export type Card = {
```

Comment:
please move all the types (one type could be one file) into one folder named models for example. there create index.js file and from there we can reference types in every other file where needed. Please adjust Card type according to the backend architecture